### PR TITLE
Update Kde packages

### PR DESCRIPTION
**Pull request by mhk2012**
- Remove duplicated packages
- Replace default phonon-gstreamer to phonon-vlc
    - phonon-vlc (this is the only maintained and supported Phonon backend for KDE apps that use Phonon for their multimedia needs; the alternative phonon-gstreamer has been unmaintained since 2013 and should not be shipped by default, or at all, ideally)
  source: https://community.kde.org/Distributions/Packaging_Recommendations

### CODE REVIEW

File: src/utils.cpp [hunk 1]
```diff
@@ -809,9 +809,9 @@ auto get_pkglist_desktop(const std::string_view& desktop_env) noexcept -> std::v
             "cachyos-kde-settings", "cachyos-themes-sddm", "cachyos-wallpapers", "char-white", "dolphin", "egl-wayland", "gwenview",
             "konsole", "kate", "kdeconnect", "kscreen", "kde-gtk-config", "khotkeys", "kinfocenter",
             "kinit", "kwallet-pam", "kwalletmanager", "plasma-wayland-protocols", "plasma-wayland-session",
-            "plasma-desktop", "plasma-framework5", "plasma-nm", "plasma-pa", "plasma-workspace", "plasma-integration",
+            "plasma-desktop", "plasma-nm", "plasma-pa", 
             "plasma-firewall", "plasma-browser-integration", "plasma-systemmonitor", "plasma-thunderbolt",
-            "powerdevil", "ksysguard", "spectacle", "sddm", "sddm-kcm", "xsettingsd", "xdg-desktop-portal", "xdg-desktop-portal-kde"};
```
Comment (vnepogodin) [inline src/utils.cpp:814]: well we would like to be explicit about installing DE packages

**Review by vnepogodin — CHANGES_REQUESTED** [review]

File: src/utils.cpp (hunk 1 above)
Comment (mhk2012) [reply src/utils.cpp:814]: I removed the commit "remove the dependency packages"

**Review by vnepogodin — APPROVED** [review]
LGTM

**Comment by vnepogodin** [comment]
Many thanks!

**Pull request merged**